moped-editor/package.json

```diff
@@ -7,7 +7,7 @@
   "private": false,
   "scripts": {
     "start": "env-cmd -e local npm run start:local",
-    "start:local": "export HTTPS=true && react-scripts start",
+    "start:local": "react-scripts start",
```

Comment:
We need HTTPS, otherwise our connections to other parts of our stack could be compromised (Hasura), or straight-up fail (Single Sign-On).

It's only matter of putting `export HTTPS=true &&` back into the start:local script.

Reply:
added it to my latest commit. I always have to remove it when I try to start the app on my local machine after syncing again with main. No worries. It's on my radar as something to fix before final commit and PR so hopefully it won't happen again.